### ISSUE TITLE
Update request

### ISSUE THREAD

**Issue by SirEnder125** [spxctreofficial/fabric-projectile-tutorial#1]
Hey. Nice tutorial. :smiley:
I was wondering if you might be willing to port the code to 1.17.1? I am trying to make a mod that adds a grenade (or a throwable TNT entity.) And I guess this tutorial teaches how to, in essence, do just that. So please consider updating this! :smiley:

**Comment by onlyspxctre** [comment]
Of course! I just got back into Fabric modding after a long hiatus, and I've realized that this projectile tutorial is severely outdated. After I have completely made sure that everything works as intended, I will update the tutorial to 1.17.

Keep in mind that I used many outdated classes when creating the initial projectile tutorial. It is highly likely I will just redirect 1.17 users to a new repository.

**Comment by SirEnder125** [comment]
Oh, thanks for actually responding!! I'm glad to hear that you actually are considering this. :D Please let me know when the new tutorial is out! Thanks again. ;)

**Comment by onlyspxctre** [comment]
Yep. I'll be working on it as a side-project though, as I have many other projects I'm currently working on (not to mention school).

**Comment by SirEnder125** [comment]
Ah, understood. :D
